gatling-tests/run-tests.ps1

```
@@ -18,57 +17,96 @@ if ($Help) {
+    Write-Host "  standard   ~3-5 minutes, balanced load for regular verification"
+    Write-Host "  extended   ~10+ minutes, sustained load for stability testing"
+    Write-Host ""
     Write-Host "Examples:"
     Write-Host "  .\run-tests.ps1"
     Write-Host "  .\run-tests.ps1 -TestProfile quick"
-    Write-Host "  .\run-tests.ps1 -TestProfile heavy -Users 50 -Duration 300"
+    Write-Host "  .\run-tests.ps1 -TestProfile extended -Users 100"
     exit 0
 }
 
+function Check-Health {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Check-Health' uses an unapproved verb.

(PSUseApprovedVerbs)